substrate/client/src/error.rs

```diff
@@ -34,6 +35,12 @@ error_chain! {
 			display("UnknownBlock: {}", &*h),
 		}
 
+		/// Extrinsic error.
+		Runtime(e: ApplyError) {
+			description("Extrinsic error"),
+			display("Extrinsic error: {:?}", e),
+		}
```

Comment:
Should we rename this to something more extrinsic-related. We could use `ApplyExtinsicFailed` for this IMO.